test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -269,6 +270,17 @@ public void TestStringEmpty()
             Assert.Null(value);
         }
 
+        [Fact]
+        public void TestDuplicatedKeys()
+        {
+            var yaml = @"
+Key1: 0
+Key1: 0
+";
+            var exception = Assert.Throws<YamlException>(() => YamlUtility.Deserialize(new StringReader(yaml)));
```

Comment:
>new StringReader(yaml) [](start = 87, length = 22)

just yaml? we have overload method to accept text

Reply:
fixed